scripts/microbe_stage/microbe_stage_hud.lua

```diff
@@ -101,6 +105,13 @@ function HudSystem:update(renderTime)
         newZVal = 80
     end
     offset.z = newZVal --]]
+
+    --make the load button activate, if and only if there actually  is a save file to load
+    if Engine:fileExists("quick.sav") then
+        self.rootGUIWindow:getChild("LoadGameButton"):enable();
+    else
+        self.rootGUIWindow:getChild("LoadGameButton"):disable();
+    end
```

Comment:
Same comment as before, we would essentially be asking the operating system 60 times a second whether this file exist.